src/pandapipes/create.py

```diff
@@ -1439,11 +1445,26 @@ def create_pipes_from_parameters(net, from_junctions, to_junctions, length_km, d
     index = _get_multiple_index_with_check(net, "pipe", index, len(from_junctions))
     _check_branches(net, from_junctions, to_junctions, "pipe")
 
+    if 'alpha_w_per_m2k' in kwargs:
+
+        if u_w_per_m2k == 0:
+            u_w_per_m2k = kwargs['alpha_w_per_m2k']
+
+        raise DeprecationWarning(' In the future alpha will be renamed to u')
```

Comment:
Here the same applies as for the comment above. The equivalence must be tested in a vectorized form. That is:

    u_zero = u_w_per_m2k == 0
    if np.any(u_zero):
        alpha = kwargs['alpha_w_per_m2k']
        if hasattr(alpha, "len"):
            u_w_per_m2k[u_zero] = np.array(list(alpha))[u_zero]
       else:
            u_w_per_m2k[u_zero] = alpha
       # now warn

I guess this should work.